src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -159,6 +154,27 @@ public static void Log(ILogger logger, IEnumerable<RestoreSummary> restoreSummar
             }
         }
 
+        private static void LogToConsole(RestoreSummary restoreSummary, LogLevel logLevel, string logHeading, LogString log)
+        {
+            var logs = restoreSummary
+                        .Errors
+                        .Where(m => m.Level == logLevel)
+                        .ToList();
+
+            if (logs.Count > 0)
+            {
+                log(string.Empty);
```

Comment:
why?

Reply:
Adds a line of separation. Not new code.

Reply:
it's okay if it's not new code, if we are improving the experience, then let's make it better.
you can add a line of separation without logging it as a warning.